.travis.yml

```diff
@@ -1,20 +1,24 @@
 language: c
 
+# Default job
+os: linux
+sudo: required
+dist: trusty
+# Should spawn a linux-gcc job and a linux-clang job
+compiler: 
+  - gcc
+  - clang
+
+# Aditional testing on OSX with default compiler: should spawn one job
 matrix:
   include:
-    - os: linux
-      sudo: required
-      dist: trusty
-      group: edge
     - os: osx
+      env: COMPILER=clang
```

Comment:
@fpetrogalli-arm Sorry, no, that was just me trying to understand what the env does and what it affects.
@shibatch I think it is generally considered bad practice to specify commands (or install packages) if it's not required. 

I will add gcc on OSX in my next patch, so we would test the following:

* Sleef on Linux with gcc-v4.8.4 (default gcc on Trusty)
* Sleef on Linux with clang-v3.5.0 (default clang on Trusty)
* Sleef on OSX with  Apple LLVM version 7.3.0 (clang-703.0.31) (default on OSX)
* Sleef on OSC with gcc-6 (needs installing) -- Are we okay with using latest version?